packages/server/src/db/utils.js

```diff
@@ -28,6 +28,7 @@ const DocumentTypes = {
   TABLE: "ta",
   ROW: "ro",
   USER: "us",
+  GROUP: "gr",
```

Comment:
This appears to be doubled up, its in the backend core and here in the server, overwriting it could be dangerous as it could end up being changed in the server but not the backend core.